src/tests/ftest/container/snapshot.py

```diff
@@ -277,6 +296,33 @@ def test_snapshot_negativecases(self):
                       "taking snapshot with a NULL epoch.")
 
 
+        #(6)DAOS-1392 Test snapshot with an invalid epoch
+        self.log.info(
+            "==(6)DAOS-1392 destroy snapshot with an invalid handle.")
+        try:
+            snapshot.destroy(None, epoch)
+            self.fail(
+                "##(6)Negative test destroy snapshot with an "
+                "invalid coh handle, expected fail, shown Passing##")
+        except Exception as error:
+            self.log.info(
```

Comment:
Can we test for the exact expected ERR to be returned when snapshot destroy fails with an invalid epoch?
Say for example it is expected to fail with ERR -1001, can we check for that and make sure it fails with the same error and not something else.

Reply:
Yes.
invalid snapshot epoch will generate Error of RC: -1002.

Reply:
Please see the newly committed script for the update.